### PR TITLE
fix: adjustments on wallet list component

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
Closes https://app.clickup.com/t/86dupz4e4
Closes https://app.clickup.com/t/86dupzb1v

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: src/domains/wallet/components/WalletsList/WalletsList.tsx [hunk 1]
```diff
@@ -174,7 +174,7 @@ export const WalletsList: React.VFC<WalletsListProperties> = ({
 								rowsPerPage={itemsPerPage}
 								currentPage={currentPage}
 								initialState={initialState}
-								className="with-x-padding"
+								className="with-x-padding after:mx-[0.15rem] after:block after:h-[5px] after:rounded-b-lg after:bg-theme-primary-100 after:content-[''] after:dark:bg-theme-secondary-800"
```
Comment (shahin-hq) [inline src/domains/wallet/components/WalletsList/WalletsList.tsx:177]: Have you tried this trick for the border bottom? https://github.com/ArdentHQ/arkvault/blob/c7f12e38a357c8caf12134e462a10a3c766a990d/src/domains/vote/components/AddressTable/AddressTable.tsx#L153

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Dropdowns are looking good.

In relation to the border, in a scenario where the "Show All" button is displayed, the table shouldn't contain the border. 

Design
image

Implementation
image

File: src/domains/wallet/components/WalletsList/WalletsList.tsx (hunk 1 above)
Comment (shahin-hq) [reply src/domains/wallet/components/WalletsList/WalletsList.tsx]: @goga-m Am I missing something?

image

**Review by samharperpittam — APPROVED** [review]

**Review by patricio0312rev — APPROVED** [review]
🔥 🔥

**Pull request merged**